doc/field_guide__cepheid.md

```diff
@@ -0,0 +1,40 @@
+## Classical Cepheids (cep)
+Classical Cepheids are young, bright (100-10,000 solar luminosities) supergiant stars that pulsate with periods of 1 to 100 days (typically several days). They are located in the main instability strip in the H-R diagram. Cepheids follow a famous pulsation period-luminosity relation, allowing the absolute magnitude of a Cepheid, and thus its distance, to be inferred from the pulsation period. Thus, Cepheids are used to measure distances to nearby galaxies.
+
+### Classification and numbers
+- Supertypes
+  - variable
+  - periodic
+  - pulsator
+- Occurrence rate: rare, about 10<sup>3</sup> expected in ZTF data, mostly in the Galactic plane and in M31
+
+
+### ZTF light curves
+![ZTF cepheids](data/cepheid_F.png)
+![ZTF cepheids](data/cepheid_F_1.png)
+![ZTF cepheids](data/cepheid_1O.png)
+![ZTF cepheids](data/cepheid_F1O.png)
+
+#### Description
+Fundamental-mode Cepheids are easy to recognise by their distinctive sawtooth light curve shape (with a rapid rise to maximum light and a slower decline) and period range (from 1 to 100 days). A secondary bump may be seen in the light curves of some Cepheids with periods in the range 6-20 days. Cepheids pulsating in the first overtone have lower amplitudes and more symmetric light curves than fundamental-mode Cepheids. Their periods (in the Milky Way) range from 0.24 to 8 days. The lower period limit is arbitrary since there is no natural boundary between first-overtone Cepheids and delta Scuti stars. Some Cepheids pulsate both in the fundamental mode and first overtone, the period ratio ranges from 0.68 to 0.77.
+
+#### Light curve characteristics
+- periodic variable
+- period range: 1-100 days (fundamental-mode pulsators), 0.24-8 days (first-overtone pulsators)
+- amplitude: moderate amplitude (~0.3 to 0.8 mag)
+- light curve shape:
+    - smooth, sawtooth variations (blue/green filters show stronger sawtooth pattern); steep rise and slow decay
+    - Classical Cepheids with pulsation periods 6-20 days may have a secondary bump
+
+#### Other characteristics and selection methods
+- intrinsic Cepheids colors: Bp-Rp = 0.8 -- 2.0 mag, but as these stars are located toward the Galactic plane, the reddening may be significant

```

Comment:
"near" the Galactic plane?

Reply:
A screenshot of what?

Reply:
Ah nevermind, I see what happened. You can delete taxonomy.html  as well. That is auto-generated too.